client/src/main/java/com/paypal/selion/internal/platform/grid/browsercapabilities/AppiumCapabilitiesBuilder.java

```diff
@@ -44,14 +45,19 @@ public DesiredCapabilities getCapabilities(DesiredCapabilities capabilities) {
         }
         capabilities.setCapability(MOBILE_NODE_TYPE, mobileSession.getMobileNodeType().getAsString());
         capabilities.setCapability(MobileCapabilityType.DEVICE_NAME, mobileSession.getDeviceType());
+        if (mobileSession.getDeviceType() != null) {
```

Comment:
Shouldn't we use `StringUtils.isNotEmpty` here instead.. Also, how about swapping lines 47 & 48 to pull 47 into this `if` block?